events/2016/fixit201606.md

```diff
@@ -4,7 +4,7 @@ Google runs internal fixit weeks. During these weeks the teams set aside all  cr
 
 Please take a look at anything that is [kind/flake] (https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20label%3Akind%2Fflake%20) or, with our 1.4 focus on "mean time to dopamine" for our users, help [team/ux] (https://github.com/kubernetes/kubernetes/labels/team%2Fux) or spend time triaging, de-duplicating, or closing issues that were [opened before 20160101] (https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=created%3A%3C2016-01-01%20state%3Aopen%20) or check out the [issues in our docs] (https://github.com/kubernetes/kubernetes.github.io/issues) repository.  All of these avenues will help Kubernetes improve the user and developer experiences with the project.
 
-Another important piece of fixit culture is rewards.  **Everyone who contributes (as measured by engagements on Github) between 6/27 and 7/1 will receive Kubernetes stickers and all merged PRs that are contributed during the fixit will will receive a Kubernetes embroidered patch.**
+Another important piece of fixit culture is rewards.  **Everyone who contributes (as measured by engagements on Github) between 6/27 and 7/1 will receive Kubernetes stickers and all merged PRs that are contributed during the fixit will receive a Kubernetes embroidered patch.**
```

Comment:
Looks like "historical" events don't need to be updated. Can you remove this change and squash the commits? Sorry for not mentioning this earlier.

Reply:
Done